### PR TITLE
RavenDB-17277 Update About Page with 5.3 License Features

### PR DESCRIPTION
**Pull request by Danielle9897**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-17277

### Additional description
Update About Page with these 5.3 license features:

Incremental TimeSeries
TCP Data Compression 
PostreSQL protocol support 
Singe-worker Data Subscriptions
Concurrent-worker Data Subscriptions

### Type of change
- New feature

### How risky is the change?
- Low 

### Backward compatibility
- Non breaking change

### Is it platform specific issue?
- No

### Documentation update
- This change requires a documentation update. 

### Testing 
- It has been verified by manual testing

### Is there any existing behavior change of other features due to this change?
- No

### UI work
- No UI work is needed

### CODE REVIEW

**Comment by arekpalinski** [comment]
Was Elasticsearch ETL already handled in a dedicated PR?

**Comment by Danielle9897** [comment]
> Was Elasticsearch ETL already handled in a dedicated PR?

yes, it was added in this commit:
`RavenDB-13041 Fixing and issue with putting empty commands when using test functionality`

and in:
`RavenDB-17147 Elastic Search ETL - Studio: Edit Task View + List View - Initial Work`

File: src/Raven.Server/Commercial/LicenseStatus.cs [hunk 1]
```diff
@@ -138,6 +138,16 @@ public int MaxClusterSize
         public bool HasPullReplicationAsSink => GetValue<bool>("pullReplicationAsSink");
 
         public bool HasTimeSeriesRollupsAndRetention => GetValue<bool>("timeSeriesRollupsAndRetention");
+        
+        public bool HasIncrementalTimeSeries => GetValue<bool>("incrementalTimeSeries");
```
Comment (ppekrol) [inline src/Raven.Server/Commercial/LicenseStatus.cs]: not needed

File: src/Raven.Server/Commercial/LicenseStatus.cs [hunk 2]
```diff
@@ -199,6 +209,11 @@ public DynamicJsonValue ToJson()
                 [nameof(HasPullReplicationAsHub)] = HasPullReplicationAsHub,
                 [nameof(HasPullReplicationAsSink)] = HasPullReplicationAsSink,
                 [nameof(HasTimeSeriesRollupsAndRetention)] = HasTimeSeriesRollupsAndRetention,
+                [nameof(HasIncrementalTimeSeries)] = HasIncrementalTimeSeries,
```
Comment (ppekrol) [inline src/Raven.Server/Commercial/LicenseStatus.cs]: not needed

File: src/Raven.Server/Commercial/LicenseStatus.cs [hunk 3]
```diff
@@ -138,6 +138,16 @@ public int MaxClusterSize
...
 
         public bool HasTimeSeriesRollupsAndRetention => GetValue<bool>("timeSeriesRollupsAndRetention");
+        
+        public bool HasIncrementalTimeSeries => GetValue<bool>("incrementalTimeSeries");
+        
+        public bool HasTcpDataCompression => GetValue<bool>("tcpDataCompression");
+        
+        public bool HasPostgreSqlProtocolSupport => GetValue<bool>("postgresSqlProtocolSupport");
+        
+        public bool HasSingleWorkerDataSubscriptions => GetValue<bool>("singleWorkerDataSubscriptions");
```
Comment (ppekrol) [inline src/Raven.Server/Commercial/LicenseStatus.cs]: not needed

File: src/Raven.Server/Commercial/LicenseStatus.cs [hunk 4]
```diff
@@ -199,6 +209,11 @@ public DynamicJsonValue ToJson()
                 [nameof(HasPullReplicationAsHub)] = HasPullReplicationAsHub,
                 [nameof(HasPullReplicationAsSink)] = HasPullReplicationAsSink,
                 [nameof(HasTimeSeriesRollupsAndRetention)] = HasTimeSeriesRollupsAndRetention,
+                [nameof(HasIncrementalTimeSeries)] = HasIncrementalTimeSeries,
+                [nameof(HasTcpDataCompression)] = HasTcpDataCompression,
+                [nameof(HasPostgreSqlProtocolSupport)] = HasPostgreSqlProtocolSupport,
+                [nameof(HasSingleWorkerDataSubscriptions)] = HasSingleWorkerDataSubscriptions,
```
Comment (ppekrol) [inline src/Raven.Server/Commercial/LicenseStatus.cs]: not needed

**Review by ppekrol — COMMENTED** [review]
@grisha-kotler please review and comment

File: src/Raven.Server/Commercial/LicenseStatus.cs (hunk 3 above)
Comment (ayende) [reply src/Raven.Server/Commercial/LicenseStatus.cs]: What is `single` here? Shouldn't that be `concurrent` ?

Comment (ppekrol) [reply src/Raven.Server/Commercial/LicenseStatus.cs]: Should be removed as I wrote. We want to add a new section in the UI called Data Subscriptions with 2 items:
- single-worker Data Subscriptions
- concurrent-worker Data Subscriptions

Comment (ppekrol) [reply src/Raven.Server/Commercial/LicenseStatus.cs]: Here: http://live-test.ravendb.net/studio/index.html#about

Comment (Danielle9897) [reply src/Raven.Server/Commercial/LicenseStatus.cs]: @ppekrol 
so I should remove `Single-worker Data Subscriptions` from here 
& simply show it in the UI **always** with the 'green check' ?

Comment (ppekrol) [reply src/Raven.Server/Commercial/LicenseStatus.cs]: correct

File: src/Raven.Server/Commercial/LicenseStatus.cs (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Server/Commercial/LicenseStatus.cs]: removed

File: src/Raven.Server/Commercial/LicenseStatus.cs (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Server/Commercial/LicenseStatus.cs]: removed

File: src/Raven.Server/Commercial/LicenseStatus.cs (hunk 4 above)
Comment (Danielle9897) [reply src/Raven.Server/Commercial/LicenseStatus.cs]: removed

File: src/Raven.Server/Commercial/LicenseStatus.cs (hunk 3 above)
Comment (Danielle9897) [reply src/Raven.Server/Commercial/LicenseStatus.cs]: removed

**Review by ml054 — APPROVED** [review]

**Pull request merged**